### DISCUSSION TITLE
Version 3.0.3.

### DISCUSSION THREAD

**Discussion by DarnokHillstar** [beabel/radiowecker#58] [Fragen & Antworten / Q&A] [answered]
Hi,

Gerade sehe ich, dass es eine neue Version gibt. Bin ein totaler Fan von Dir (Eigentlich von der kompletten Comunity, die dieses Projekt so weit gebracht hat - Dank somit auch an Hr. Lechner).

Blöde Frage: Was ist an der 3.0.3. Version geändert worden. Im ersten Schuss, nach dem Hochladen habe ich jetzt nichts groß unterschiedliches entdeckt. Bitte entschuldige, wenn ich den Code im Detail noch nicht durchgegangen bin.

Kannst diese "Discussion" von mir gerne löschen / ignorieren, wenn es nachgetragen wird.

Ein großer Fan.

DarnokHillstar

**Comment by beabel** [comment, marked as answer]
Die Änderungen sind [hier](https://github.com/beabel/radiowecker/commit/fb09a7f56ed0a5f36bf4f83415ad26b91ccc5f6d) aufgelistet.

Im großen und ganzen ist nicht viel geändert worden eher "aufräumen und strukturieren". Zumindest ist nichts "sichtbares" neu ;-)
Hauptsächlich ist neu: 
1. Automatische Touch Drehung bei Set. Rotation des Displays von 3 -> 1
2. Die Arduino JSON Lib wurde aktualisiert
3. in der audio.ino wurde die MDCallback mit etwas mehr Debug ausgaben erweitert
4. der Code wurde einheitlich formatiert 

Gerade in Arbeit (in der dev_304) ist (auch wieder "aufräumen und strukturieren" + neue Features als Idee: 
1. die komplette Code Kommentierung -> fertig
1.1->dies war mir wichtig, damit auch andere und auch ich später schneller wieder in den Code reinfinde
3. bessere Aufteilung in mehr einzelne Tabs -> in Arbeit
4. Versuch einer Möglichkeit direkt von GitHub ein Update zu starten -> bisher nur Idee
5. neue Seite auf dem Screen "Info" ähnlich wie im Web Interface -> bisher nur Idee
6. Aufteilung der tft_display Datei in Dateien je Screen um einfacher erweitern zu können -> bisher nur Idee
 

[Hier ](https://github.com/beabel/radiowecker/compare/main...develop_304)sieht man was gerade als Unterschied zur aktuellen Version vorhanden ist

**Comment by fraeggle** [comment]
Hi
In deinem Online Flasher hast du die aber noch nicht, oder ? Weil da seh ich nur als letztes die 3.0.2

Gruß Peter

**Reply by beabel** [reply]
Sorry, habe ich natürlich sofort erledigt.

Sollte jetzt verfügbar sein

**Comment by fraeggle** [comment]
:-O ich wollte dich nicht unter Stress setzen  ;-)

Danke schön...
